tox.ini

```diff
@@ -103,7 +103,7 @@ deps =
     protobuf==3.20.3
     pydantic
     pytest
-    juju==3.3.1.1
+    juju==3.6.1.0
     pytest-operator
     pytest-asyncio
     websockets<14.0 # https://github.com/juju/python-libjuju/issues/1184
```

Comment:
Hasn't this been fixed alraedy?

Reply:
I'm not sure about that, but I also don't mind pinning pylibjuju to a more recent version.